src/assets/style.css

```diff
@@ -17,11 +17,8 @@
   min-height: 100%;
 }
 
-.playkit-fullscreen .playkit-bumper-container {
-  height: 100%;
-}
-
 .playkit-bumper-container video {
+  position: absolute;
```

Comment:
I see that .embed-responsive set position absolute for video tag, doesn't it get position absolute anyway?

Reply:
what is `.embed-responsive`?

Reply:
doesn't matter, I saw it on tests pages